package.json

```diff
@@ -1,68 +1,86 @@
+    "flatten:core": "npm run flatten contracts/core/core/",
     "flatten:libs": "npm run flatten contracts/libs/WitnetErrorsLib.sol && npm run flatten contracts/libs/WitnetEncodingLib.sol && npm run flatten contracts/libs/WitnetPriceFeedsLib.sol",
-    "flatten:proxy": "npm run flatten contracts/impls/WitnetProxy.sol",
+    "flatten:proxy": "npm run flatten contracts/core/WitnetProxy.sol",
     "fmt:js": "eslint \"**/*.js\"",
     "fmt:sol": "solhint --max-warnings 0 \"contracts/**/*.sol\" && solhint \"test/**/*.sol\" && solhint \"flattened/**/*.sol\"",
     "fmt!:js": "eslint \"**/*.js\" --fix",
     "fmt!:sol": "solhint --max-warnings 0 \"contracts/**/*.sol\" --fix && solhint \"test/**/*.sol\" --fix && solhint \"flattened/**/*.sol\"",
     "fmt!": "npm run fmt!:js && npm run fmt!:sol",
     "fmt": "npm run fmt:js && npm run fmt:sol",
-    "migrate": "node ./scripts/migrate.js 2>&1",
+    "migrate": "npx truffle migrate --network",
     "networks": "node ./scripts/networks.js 2>&1",
-    "ops:rng:sla": "truffle migrate --migrations_directory ./migrations/ops/rng/sla --network",
-    "test": "npm run clean && truffle test"
+    "ops:rng:sla": "npx truffle migrate --migrations_directory ./migrations/ops/rng/sla --network",
+    "prepare": "npx truffle compile --all && npx hardhat compile && node ./scripts/prepare.js",
+    "test": "npm run clean && npx truffle test"
   },
-  "author": "Witnet Foundation <info@witnet.foundation>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/witnet/witnet-solidity-bridge.git"
-  },
-  "keywords": [
-    "bridge",
-    "witnet",
-    "ethereum"
-  ],
-  "license": "MIT",
   "dependencies": {
-    "@openzeppelin/contracts": "4.8.0",
-    "@openzeppelin/contracts-upgradeable": "4.8.0",
-    "ado-contracts": "1.0.0",
-    "lodash": "4.17.21"
+    "dotenv": "^16.4.1",
+    "lodash.merge": "^4.6.2",
+    "proper-lockfile": "^4.1.2"
   },
   "devDependencies": {
+    "@nomicfoundation/hardhat-verify": "^2.0.4",
+    "@openzeppelin/contracts": "^5.0.1",
+    "@openzeppelin/contracts-upgradeable": "^5.0.1",
     "@openzeppelin/test-helpers": "~0.5.16",
-    "chai": "4.3.6",
+    "ado-contracts": "1.0.0",
+    "bn.js": "^4.11.0",
     "custom-error-test-helper": "^1.0.6",
-    "dotenv": "8.2.0",
-    "eslint": "~8.31.0",
-    "eslint-config-standard": "~17.0.0",
-    "eslint-plugin-import": "~2.26.0",
-    "eslint-plugin-n": "~15.6.0",
-    "eslint-plugin-promise": "~6.1.1",
-    "eth-create2": "~1.0.1",
-    "eth-gas-reporter": "0.2.25",
-    "js-sha256": "0.9.0",
-    "solhint": "3.3.7",
-    "solidity-coverage": "0.7.16",
-    "truffle": "~5.11.5",
-    "truffle-assertions": "0.9.2",
-    "truffle-flattener": "~1.6.0",
-    "truffle-plugin-verify": "~0.6.2"
+    "eslint": "^8.56.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-promise": "^6.1.1",
```

Comment:
I think we aren't using this `eslint` files but `eslint-config-standard`, could we remove the other ones?